src/html/partials/question.html

```diff
@@ -1,4 +1,16 @@
 <section class="question-section">
-  <h2 class="sm-title">The Question</h2>
-  <p class="bg-paragraph">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</p>
+
+  <div class="wrapper-title__question">
+    <h2 class="sm-title">The Question</h2>
+    <p class="bg-paragraph">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</p>
```

Comment:
What does the class "bg-paragraph" stand for?

Reply:
![screen shot 2019-01-30 at 3 33 12 pm](https://user-images.githubusercontent.com/28491570/52010796-6341bd80-24a4-11e9-8f6b-04086b30019c.png)

The sections under the h2 are consistently styled the same so I'm calling them "big" or bg-paragraph.

Reply:
Gotcha. Another naming option to consider is something with the word "intro" in it, for introduction.